spec/fixtures/external_certs/badalt.crt

```diff
@@ -0,0 +1,72 @@
+Certificate:
```

Comment:
It is notably interesting that the system accepts certificates with the clean textual version of the certificate at the top. This is a common pattern for certificates but I had previously thought that this is not allowed. The documentation on what is acceptable may need to be adjusted to mention this.